### PR TITLE
Plugin file watcher only reminds user to restart node

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Replace https://github.com/neo-project/neo/pull/2569

### CODE REVIEW

File: src/Neo/Plugins/Plugin.cs [hunk 1]
```diff
@@ -102,20 +103,8 @@ private static void ConfigWatcher_Changed(object sender, FileSystemEventArgs e)
...
-                    break;
                 case ".dll":
-                    if (e.ChangeType != WatcherChangeTypes.Created) return;
-                    if (GetDirectoryName(GetDirectoryName(e.FullPath)) != PluginsDirectory) return;
-                    try
-                    {
-                        LoadPlugin(Assembly.Load(File.ReadAllBytes(e.FullPath)));
-                    }
-                    catch { }
+                    Utility.Log(nameof(Plugin), LogLevel.Warning, $"File {e.Name} is {e.ChangeType}, please restart node.");
```
Comment (erikzhang) [inline src/Neo/Plugins/Plugin.cs:107]: If multiple files are added or deleted, many logs will be printed.

Comment (superboyiii) [reply src/Neo/Plugins/Plugin.cs:107]: Multi log is normal, better than multi loading. Delete all is not a normal behaviour when node is running.

**Review by erikzhang — DISMISSED** [review]

**Comment by nicolegys** [comment]
1. missing `configWatcher.Deleted`
2. too many logs when adding a plugin
image

**Comment by Ashuaidehao** [comment]
>     2. too many logs when adding a plugin
>        image

It depends on file system, reference to  https://learn.microsoft.com/en-us/dotnet/api/system.io.filesystemwatcher.created?redirectedfrom=MSDN&view=net-6.0

**Review by nicolegys — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**